### PR TITLE
Update template manifest

### PR DESCRIPTION
**Pull request by PerVillalva**

### CODE REVIEW

**Review by jbartadev — COMMENTED** [review]
Hey Perci @PerVillalva, I updated the manifest with the new properties [we need](https://github.com/apify/apify-web/issues/2343) for the new template library in the console - I added `label`, `category`, and `technologies`. Because of these changes the descriptions still look quite poor and not very useful. Could someone from marketing do another iteration of these so it's usable the way it is presented in [the new design](https://www.figma.com/proto/oVEMTguG3LLMyHgw9YcroE/Creating-Actor---uplift?node-id=1330%3A78844&scaling=min-zoom&page-id=403%3A60&starting-point-node-id=1016%3A18446&show-proto-sidebar=1) we're currently working on?

**Comment by PerVillalva** [comment]
@jbartadev Thanks for the updated structure! It does look more comprehensive and well-organized than before. I've made a few adjustments to the descriptions so they are similar to the new text we have in the console.

Please let me know if you think the descriptions are better now, and then I can ask Theo to proofread them to ensure everything is correct.

File: templates/manifest.json
Comment (mnmkng) [inline templates/manifest.json:31]:
```suggestion
            "label": "Start with Python",
```

File: templates/manifest.json
Comment (jbartadev) [inline templates/manifest.json:21]: `using TypeScript and Node.js.` - the `Using` should have the capital letters as it is at the beginning of the sentence

```suggestion
            "description": "Basic template for developing actors on the Apify platform. Using TypeScript and Node.js.",
```

File: templates/manifest.json
Comment (jbartadev) [inline templates/manifest.json:33]: Same here - I'm also not sure if `Using python` sounds and looks good as a sentence 😸 
```suggestion
            "description": "Basic template for developing actors on the Apify platform. Using Python.",
```

**Review by jbartadev — CHANGES_REQUESTED** [review]
Thanks, Perci for the quick changes and reaction! I guess it's much better now. I just found small typos in the description. 🚀

Theo makes it perfect for sure, please let me know when is he going to be able to check it so we can move forward 😄

**Comment by PerVillalva** [comment]
@jbartadev  I will contact Theo today and ping you once he finishes the text's final proofreading.

File: templates/manifest.json
Comment (TheoVasilis) [inline templates/manifest.json:6]: platform. using - remove .

File: templates/manifest.json
Comment (TheoVasilis) [inline templates/manifest.json:21]: Basic template for developing actors on the Apify platform using TypeScript and Node.js.

File: templates/manifest.json
Comment (TheoVasilis) [inline templates/manifest.json:46]: Standard and up-to-date template for developing with Crawlee's CheerioCrawler.

File: templates/manifest.json
Comment (TheoVasilis) [inline templates/manifest.json:60]: Standard and up-to-date template for developing with Crawlee's PuppeteerCrawler.

File: templates/manifest.json
Comment (TheoVasilis) [inline templates/manifest.json:73]: Standard and up-to-date template for developing with Crawlee's PlaywrightCrawler.

File: templates/manifest.json
Comment (TheoVasilis) [inline templates/manifest.json:86]: Standard and up-to-date template for developing with Crawlee's CheerioCrawler.

File: templates/manifest.json
Comment (TheoVasilis) [inline templates/manifest.json:100]: Standard and up-to-date template for developing with Crawlee's PuppeteerCrawler.

File: templates/manifest.json
Comment (TheoVasilis) [inline templates/manifest.json:113]: Standard and up-to-date template for developing with Crawlee's PlaywrightCrawler.

**Review by TheoVasilis — CHANGES_REQUESTED** [review]

**Review by jbartadev — APPROVED** [review]
Nice! Looks good to me. Thanks, guys!

**Comment by jbartadev** [comment]
@PerVillalva Just a note: Please do not merge it yet as it could break some of the current logic of the templates in the console (the new version is in progress)

**Comment by PerVillalva** [comment]
@jbartadev Alright, I will leave the decision to merge up to you then. Just ping me if you need anything else 🤝

**Review by TheoVasilis — APPROVED** [review]

**Comment by HonzaTuron** [comment]
> @PerVillalva Just a note: Please do not merge it yet as it could break some of the current logic of the templates in the console (the new version is in progress)

@jbartadev New json seems to be backward-compatible with the old one (just added fields, right @PerVillalva?)

**Comment by jbartadev** [comment]
> > @PerVillalva Just a note: Please do not merge it yet as it could break some of the current logic of the templates in the console (the new version is in progress)
> 
> @jbartadev New json seems to be backward-compatible with the old one (just added fields, right @PerVillalva?)

Yes it is in a way @HonzaTuron - but as you can see we're parsing the description to get the template title in the console and in this PR we added `title` field and remove the prefix from the `description` so the current logic wouldn't work properly. Make sense?

See 
Snímek obrazovky 2023-02-06 v 11 34 37

**Comment by HonzaTuron** [comment]
Please do not remove this branch after merging as production console mistakenly pointing to this branch. But feel free to merge. Thanks!

**Pull request merged**
